src/azure-cli/azure/cli/command_modules/synapse/commands.py

```diff
@@ -120,5 +124,18 @@ def get_custom_sdk(custom_module, client_factory):
         g.custom_show_command('show', 'get_spark_session_statement')
         g.custom_command('cancel', 'cancel_spark_session_statement', confirmation=True)
 
+    # Data Plane Commands --Access control operations
+    with self.command_group('synapse role assignment', synapse_accesscontrol_sdk,
+                            custom_command_type=get_custom_sdk('accesscontrol', None)) as g:
+        g.custom_command('create', 'create_role_assignment')
+        g.custom_command('list', 'list_role_assignments')
+        g.custom_show_command('show', 'get_role_assignment_by_id')
+        g.custom_command('delete', 'delete_role_assignment_by_id', confirmation=True)
+
+    with self.command_group('synapse role definition', synapse_accesscontrol_sdk,
+                            custom_command_type=get_custom_sdk('accesscontrol', None)) as g:
+        g.custom_command('list', 'list_role_definitions')
```

Comment:
No other operations (create, delete)?

Reply:
Yes, create/delete role definition operation is not supported.